integration_test/pg/constraints_test.exs

```diff
@@ -33,6 +32,7 @@ defmodule Ecto.Integration.ConstraintsTest do
   end
 
   setup_all do
+    PoolRepo.delete_all(Ecto.Migration.SchemaMigration)
```

Comment:
Why is this necessary? Nothing else should have ran the migrations in this file, right?

Reply:
Somehow I got a warning when running `ECTO_ADAPTER=pg mix test` saying that a migration with a higher timestamp has been run (in `integration_tests/sql/migration.exs`) 🤔. Did I miss out anything?

Reply:
I deleted the line and fortunately [this Travis job](https://travis-ci.org/elixir-ecto/ecto/jobs/435019781) has the warning I mention.

Reply:
Oh, I see. Leave the warning on and async: true, we will find a way to fix it. :)